### DISCUSSION TITLE
More options for element colors, to help with colorcoding

### DISCUSSION THREAD

**Discussion by Oreztov** [diesel-modding/BeardLib-Editor#618] [Ideas]
Currently, the only two options for the colors of elements are random colors or one uniform color. That's the exact opposite of helpful, as both options don't help with sorting out elements or making them recognizable from one another.
**The suggestion is to add an option to individually change the color of every element type, making sorting elements much easier.**
Note: I'm not the only one who wants this to be done, multiple people over at payday2maps.net have discussed this.

**Comment by MiamiCenterPL** [comment]
Another similar suggestion would be to set element icons depending on what is the element.
I presonally think there should be a shape for elements depending on a type (every Unit-related element could be Quadrant, every trigger would be rhombus etc.)
